plugins/GSdx/GSTextureCache.cpp

```diff
@@ -480,6 +480,10 @@ GSTextureCache::Target* GSTextureCache::LookupTarget(const GIFRegTEX0& TEX0, int
 			GSVector2i custom_resolution = m_renderer->GetCustomResolution();
 			scale_factor.x = static_cast<float>(custom_resolution.x) / width;
 			scale_factor.y = static_cast<float>(custom_resolution.y) / height;
+
+			//Don't bother setting a scale factor when people request same resolution as native resolution.
+			if(custom_resolution == GSVector2i(width, height))
+				scale_factor = GSVector2(0,0);
```

Comment:
@refractionpcsx2 

```
if(custom_resolution == GSVector2i(width, height))
				scale_factor = GSVector2(0,0);
```

what's wrong with the current code ? It's fairly simple to understand and looks more clean over something like,
 ``if (scale_factor.x && scale_factor.y && (scale_factor.x !=1 && scale_factor.y!=1))``